modules/clipboard/init.zsh

```diff
@@ -0,0 +1,62 @@
+#
+# Integrates the zsh clipboard with the system clipboard.
+#
+# Authors:
+#   Nir Friedman <quicknir@gmail.com>
+#
+
+# If running Mac OS, set this to 1
+local IS_MAC_OS=0
+
+if [[ IS_MAC_OS -eq 0 ]]; then
```

Comment:
This means the if check will be taken every single time the function is called though. I guess we don't care about this cost?